### PR TITLE
Beach Planet

### PR DESCRIPTION
**Pull request by Jackriip**
atvs jukeboxes lots of booze a gun arachnids slimes multi-biome custom tendrils ships yadadadada

### CODE REVIEW

**Comment by ShibaInuLord** [comment]
wouldnt it be cooler if the serpent tails were called kraken tentacles instead or something

**Comment by ShibaInuLord** [comment]
~~another idea- maybe rename the planet to ocean planet to better fit the naming convention?~~
edit: it doesnt really matter actually lmao

**Comment by SuperNovaa41** [comment]
Your maps should be saved in TGM mode. (Open in SDMM -> Prefernces -> saving mode: TGM -> Save)

File: voidcrew/code/datums/mapgen/BeachGenerator.dm
Comment (SuperNovaa41) [inline voidcrew/code/datums/mapgen/BeachGenerator.dm]:
```suggestion
	for(var/turf in turfs) //Go through all the turfs and generate them
```

File: voidcrew/code/datums/mapgen/BeachGenerator.dm
Comment (SuperNovaa41) [inline voidcrew/code/datums/mapgen/BeachGenerator.dm]:
```suggestion
		var/area/area = gen_turf.loc //meet my friends, Ctrl+C and Ctrl+V!
```

**Review by SuperNovaa41 — CHANGES_REQUESTED** [review]
Ok so theres a couple more of these, fix all the uhh single letter var names pls :)

**Comment by Jackriip** [comment]
Continuing to add to this, don't merge it

**Comment by SuperNovaa41** [comment]
Even better we should wait until my planet refactor is merged before we merge this :)

File: code/__DEFINES/sound.dm
Comment (SuperNovaa41) [inline code/__DEFINES/sound.dm:106]: move this into `code/__DEFINES/__VOID_DEFINES/sound.dm`

File: shiptest.dme
Comment (SuperNovaa41) [inline shiptest.dme:3182]: these are all placed in the wrong place.. redo this and put them in the proper alphabetical spot. Dreammaker likes to force rearrange these and i'd rather have them all done here so someone else's PR doesn't throw this in

File: voidcrew/code/datums/ruins/beach.dm
Comment (SuperNovaa41) [inline voidcrew/code/datums/ruins/beach.dm:1]: make a generic `/datum/map_template/ruin/beach/` and put the prefix there, so you don't have to include it with every single ruin

File: voidcrew/code/game/objects/structures/beach/serpent_tail.dm
Comment (SuperNovaa41) [inline voidcrew/code/game/objects/structures/beach/serpent_tail.dm:16]: make sure to put a comment with `///` on the line before these and document what they do

File: voidcrew/code/game/objects/structures/beach/serpent_tail.dm
Comment (SuperNovaa41) [inline voidcrew/code/game/objects/structures/beach/serpent_tail.dm:21]: no single letter variables ALLOWED

File: voidcrew/code/modules/mob/living/simple_animal/friendly/beachcarp.dm
Comment (SuperNovaa41) [inline voidcrew/code/modules/mob/living/simple_animal/friendly/beachcarp.dm:26]: `/mob/living/simple_animal/voidcarp` doesn't exist. one of these two paths is wrong

File: voidcrew/code/modules/vehicles/atv.dm
Comment (SuperNovaa41) [inline voidcrew/code/modules/vehicles/atv.dm:2]:
```suggestion
	name = "atv"
```
It's in the description

File: voidcrew/code/modules/vending/boozeomat.dm
Comment (SuperNovaa41) [inline voidcrew/code/modules/vending/boozeomat.dm:4]:
```suggestion
	desc = "A conventiently placed Booze-O-Mat, seems to have free drinks!"
```
bike shedding but like im sure most people don't care about if it uses power, they will just see funny drink machine

**Review by SuperNovaa41 — CHANGES_REQUESTED** [review]
A lot of this looks really good! just a few things need changing

**Pull request closed without merging**
